### PR TITLE
docs: update qgb evm step of validator-node.mdx

### LINKED ISSUE #968: Update the docs for `--evm-address`
## Context

celestia-app [v1.0.0-rc12](https://github.com/celestiaorg/celestia-app/releases/tag/v1.0.0-rc12) contains a breaking change that removes the `--evm-address` flag. 

## Problem

The docs still reference the `--evm-address` flag. See https://docs.celestia.org/nodes/validator-node/#setup-qgb-keys

## Proposal

Update the docs to remove the `--evm-address` flag and describe the new mechanism for sending a tx to register a validator's EVM address. See https://github.com/celestiaorg/celestia-app/commit/8428940f9df4f0411bd5d0385a244f87ec743aa6

We may need to preserve the docs for the `--evm-address` flag for releases prior to v1.0.0-rc12. Defer to @SweeXordious

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #968 

## Checklist

- [x] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/validator-node.mdx [hunk 1]
```diff
@@ -325,7 +350,90 @@ You have the following option of connecting to list of networks shown below:
...
+timestamp: ""
+tx: null
+txhash: <tx-hash>
+```
+
+## Register your validator's EVM address
+
+This section will cover how to register your validator's EVM address.
+This is required if you run an orchestrator. If you don't run an
+orchestrator, you don't need to register your EVM address.
```
Comment (SweeXordious) [inline docs/nodes/validator-node.mdx:397]:
```suggestion
This is required to run an orchestrator.
```

I guess it's better not to show that validators have a choice not to run the orchestrator

File: docs/nodes/validator-node.mdx [hunk 2]
```diff
@@ -325,7 +350,90 @@ You have the following option of connecting to list of networks shown below:
...
+
+To register your EVM address, run the following. Be sure to replace
+`YOUR_EVM_ADDRESS` with your EVM address:
+
+```bash
+VALIDATOR_ADDRESS=$(celestia-appd keys show validator --home "${HOME_DIR}" --bech val -a)
```
Comment (SweeXordious) [inline docs/nodes/validator-node.mdx:403]: `validator` depends on the name they used to create their keys. Shouldn't it be `$VALIDATOR_WALLET` ?

File: docs/nodes/validator-node.mdx [hunk 3]
```diff
@@ -325,7 +350,90 @@ You have the following option of connecting to list of networks shown below:
...
+EVM_ADDRESS="YOUR_EVM_ADDRESS"
+
+celestia-appd tx qgb register \
+    "${VALIDATOR_ADDRESS}" \
+    "${EVM_ADDRESS}" \
+    --from validator \
```
Comment (SweeXordious) [inline docs/nodes/validator-node.mdx:409]: same

**Review by SweeXordious — COMMENTED** [review]
Thanks a lot for helping with this :+1:
 
Left some comments

File: docs/nodes/validator-node.mdx [hunk 4]
```diff
@@ -325,7 +350,89 @@ You have the following option of connecting to list of networks shown below:
...
```
Comment (jcstein) [inline docs/nodes/validator-node.mdx:408]:
```suggestion
    --from $VALIDATOR_WALLET \
```

**Review by SweeXordious — APPROVED** [review]
utAck.

Thanks a lot for handling this. Approving given merge conflicts are handled

**Comment by jcstein** [comment]
you're welcome, thanks for the help getting it straightened out. i aim to test this tomorrow or friday

**Comment by jcstein** [comment]
note to also update all qgb-deploy/intro/keys/orchestrator/relayer in alignment with what is in current https://github.com/celestiaorg/orchestrator-relayer/tree/main/docs

**Pull request merged**